rust/sbd-server/src/test.rs

```diff
@@ -93,7 +93,7 @@ async fn auth_with_real_token_provider() {
         auth_material: Some(b"hello".to_vec()),
         alter_token_cb: Some(Arc::new(|t| {
             println!("TEST GOT TOKEN: {t}");
-            assert_eq!(r#"{"authToken":"bob"}"#, &*t);
+            assert_eq!("bob", &*t);
```

Comment:
Oh boy. I even had a test for this already... it just didn't seem wrong to me to have the full json here when I was writing it :facepalm:

Reply:
It was tested, that's the main thing. It was doing what it had been verified to do :D 

Have done this, quite a few times - it happened with the countersigning stuff and with gossip :D